resources/update-protocols.sh

```diff
@@ -12,8 +12,8 @@ declare -a REPOS=(
     "core          https://gitlab.freedesktop.org/wayland/wayland.git"
     "standard      https://gitlab.freedesktop.org/wayland/wayland-protocols.git"
     "mir           https://github.com/MirServer/mir.git"
-    "wlroots       https://github.com/swaywm/wlroots.git"
-    "wlr-protocols https://github.com/swaywm/wlr-protocols.git"
+    "wlroots       https://gitlab.freedesktop.org/wlroots/wlroots.git"
```

Comment:
The protocol used by wlroots is duplicated with wlr-protocols. Should this be deleted?

Reply:
Looks like `wlr-input-inhibitor-unstable-v1.xml` is only in wlr-protocols and `drm.xml` is only in wlroots, so keep both